charts/app-config/values-integration.yaml

```diff
@@ -2377,6 +2388,15 @@ govukApplications:
         value: 759acac6-da53-4a19-b591-b7538c7c39de
       - name: REDIS_URL
         value: redis://shared-redis-govuk.eks.integration.govuk-internal.digital
+    nginxConfigMap:
+      extraConf: |
+        client_max_body_size 500m;
```

Comment:
nit: best to write this as M rather than m for least surprise. While nginx treats both as mega, in k8s m is milli and M is mega.